OMCompiler/Compiler/NBackEnd/Modules/2_Pre/NBAlias.mo

```diff
@@ -1139,23 +1175,8 @@ protected
           BackendExtension.VariableAttributes attr;
 
       case Variable.VARIABLE(backendinfo=BackendExtension.BACKEND_INFO(attributes=attr as BackendExtension.VariableAttributes.VAR_ATTR_REAL())) algorithm
-
-        if Util.isSome(attr.min) then
-          min_val := Util.getOption(attr.min);
-          UnorderedMap.add(BVariable.getVarName(var_ptr), min_val, attrcollector.min_val_map);
-        end if;
-        if Util.isSome(attr.max) then
-          max_val := Util.getOption(attr.max);
-          UnorderedMap.add(BVariable.getVarName(var_ptr), max_val, attrcollector.max_val_map);
-        end if;
-        if Util.isSome(attr.start) then
-          start_val := Util.getOption(attr.start);
-          UnorderedMap.add(BVariable.getVarName(var_ptr), start_val, attrcollector.start_map);
-        end if;
-        if Util.isSome(attr.fixed) then
-          fixed_val := Util.getOption(attr.fixed);
-          UnorderedMap.add(BVariable.getVarName(var_ptr), fixed_val, attrcollector.fixed_map);
-        end if;
+        attrcollector := optionMinMax(var_ptr, attr.min, attr.max, attrcollector);
+        attrcollector := optionMinMax(var_ptr, attr.start, attr.fixed, attrcollector);
```

Comment:
this second call of  `optionMinMax()` (and the one in `VAR_ATTR_INT()`) is supposed to be `optionStartFixed()` right?